config/my.cnf

```diff
@@ -56,13 +57,17 @@ innodb_log_buffer_size=8M
 innodb_file_per_table=1
 innodb_stats_on_metadata=0
 
-# Longer timeouts for datajoint populate jobs, workgroup lan interactivity
+# Longer timeouts for datajoint populate jobs, workgroup lan interactivity, rollback on timeout
```

Comment:
While we're at it, can you also explicitly add ONLY_FULL_GROUP_BY to the 'SQL MODE' section?
was a long standing item to add per conversation elsewhere that I didn't get to -

In current versions it defaults to 'on' so this is a bit redundant, however it's good to note that we explicitly rely on this 
since it can be disabled and at least for now, the configuration here is the 'reference' configuration that we point people to when configuring servers.

https://dev.mysql.com/doc/refman/5.7/en/group-by-handling.html
https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#sqlmode_only_full_group_by

Reply:
Ah yes, I forgot about this discussion on Slack. Updated.